### PR TITLE
Adding readme

### PR DESCRIPTION
**Pull request by geramirez**
- facilitate the creation of .env file
- document docker-compose command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+## Software Requirements 
+* docker
+* docker-compose
+
+## Setup
+
+### 1. Create .env file with keys
+```bash 
+cat << EOF > .env
+CASEID_ENCRYPTION_KEY=$(echo -n date | base64)
```
Comment (cailyncodes) [inline README.md]: `attr_encrypted` requires that the key be 32 bytes (256-bit). I'm not sure `date` will give us enough. So I think we might need something more like this `$(cat /dev/random | dd ibs=32 count=1 status=none | base64)`?

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+CELLPHONENUMBER_ENCRYPTION_KEY=$(echo -n date | base64)
```
Comment (cailyncodes) [inline README.md]: Same as above

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
\ No newline at end of file
+# Care-4-Kids Auto Notifier
+
+## Software Requirements 
+* docker
+* docker-compose
+
+## Setup
+
+### 1. Create .env file with keys
```
Comment (cailyncodes) [inline README.md]: I like the how you were explicit here and give examples to generate keys!

File: README.md (hunk 1 above)
Comment (geramirez) [reply README.md]: ! I just realized this when trying to save something to the applicant model, I like your solution.

**Comment by cailyncodes** [comment]
Following current standard of no encryption on caseid

**Pull request merged**